File: dev-docs/snippets/auth/otp-request.swift
import JsBaoClient

// Email a one-time password (OTP) code to the user via the typed `client.auth`
// namespace. Returns a typed `OtpRequestResult` (`success`).
func otpRequest(client: JsBaoClient, email: String) async throws {
  // #region example
  let result = try await client.auth.otpRequest(email: email)
  let success = result.success
  // #endregion example
  _ = success
}

File: dev-docs/snippets/auth/otp-request.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Email a one-time password (OTP) code to the user.
export async function otpRequest(client: JsBaoClient, email: string) {
  // #region example
  const { success } = await client.otpRequest(email);
  // #endregion example
  return success;
}
